workspaces/spectacle/src/graphql/schema.ts

```diff
@@ -8,6 +8,7 @@ type Mutation {
   applyCommands(commands: [JSON!]!, batchCommitId: ID!, commitMessage: String!, clientId: ID!, clientSessionId: ID!): AppliedCommandsResult
   startDiff(diffId: ID!, captureId: ID!): StartDiffResult
   invalidateCaches: InvalidateCachesResult
+  resetToCommit(batchCommitId: ID!): Boolean
```

Comment:
We aren't returning anything here, GQL doesn't have a way to define explicit null types. The omission of a `!` means we can return null. 

There isn't much too useful to return here (apart from whether there was a batchCommit removed - but that's not likely to be super useful given you are likely querying batch commits -> calling this once you have a list of valid commits which is why I just returned null.